utils/api/benefits/oasBenefit.ts

```diff
@@ -71,7 +71,7 @@ export class OasBenefit extends BaseBenefit<EntitlementResultOas> {
               ? this.translations.detail.oas.eligibleIncomeTooHigh
               : this.translations.detail.eligible,
         }
-      } else if (this.input.age === 64) {
+      } else if (this.input.age >= 64 && this.input.age < 65) {
```

Comment:
isn't the same message if  just  < 65  ???

Reply:
no, they are not same